### DISCUSSION TITLE
Why is index.js source code displayed instead of "Hello world" when using a subfolder?

### DISCUSSION THREAD

**Discussion by oeloo** [vercel/community#7252] [Help] [answered]
### Summary

Hi, I have created a basic Hello world example that works as expected when at the root of my GitHub repository:
![image](https://github.com/vercel/community/assets/98819304/4bc00841-e587-43a7-b1ef-3140a0673cfc)
Here is `versel.json `
```
{
  "version": 2,
  "builds": [
    {
      "src": "index.js",
      "use": "@vercel/node"
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "/index.js"
    }
  ]
}
```
it is working as expected: 
![image](https://github.com/vercel/community/assets/98819304/5c494474-5757-4b89-a75d-2989d219cf0d)

The problem arises when placing this exact same `vercel.json` and `index.js` in a subfolder (named `backend`):
![image](https://github.com/vercel/community/assets/98819304/d3fdc54c-d0e4-42ff-beec-85da65eddfb9)
I get this: 
![image](https://github.com/vercel/community/assets/98819304/f24a01c0-82e1-4f7c-a365-b825576a7913)
index.js source code is displayed instead of "Hello World" even though I had configured this Vercel project with `backend `as root directory:
![image](https://github.com/vercel/community/assets/98819304/585f3977-41ae-46f2-aad1-018e7d5490e2)

### Steps to Reproduce

I have provided them when explaining the issue.

**Comment by versecafe** [comment, marked as answer]
@oeloo In your second example you have a typo as `versel.json` so it's not doing anything just rename the file

**Reply by oeloo** [reply]
Thanks, that was the issue. I was looking too much at the configurations and not checking enough filenames.

**Reply by versecafe** [reply]
Happens to us all tiny mistake that sneaks through

[a comment by an automated account is not shown]
